samples/drivers/crypto/sample.yaml

```diff
@@ -51,6 +51,22 @@ tests:
         - ".*: CBC mode DECRYPT - Match"
         - ".*: CTR mode ENCRYPT - Match"
         - ".*: CTR mode DECRYPT - Match"
+  sample.drivers.crypto.stm32l4:
+    tags: crypto
+    filter: dt_compat_enabled("st,stm32l4-aes")
+    integration_platforms:
+      - nucleo_l4a6zg
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - ".*: Cipher Sample"
+        - ".*: ECB mode ENCRYPT - Match"
+        - ".*: ECB mode DECRYPT - Match"
+        - ".*: CBC mode ENCRYPT - Match"
+        - ".*: CBC mode DECRYPT - Match"
+        - ".*: CTR mode ENCRYPT - Match"
+        - ".*: CTR mode DECRYPT - Match"
```

Comment:
This is not required. What you just need to do is enable aes node directly in `nucleo_l4a6zg.dts` and then it will match filter of already existing test config `sample.drivers.crypto.stm32` thanks to `dt_compat_enabled("st,stm32-aes")`.